src/battle_script_commands.c

```diff
@@ -16350,7 +16350,7 @@ void BS_JumpIfEmergencyExited(void)
 void BS_JumpIfRod(void)
 {
     NATIVE_ARGS(const u8 *jumpInstr);
-    if (IsElectricAbilityAffected(ABILITY_LIGHTNING_ROD))
+    if (B_REDIRECT_ABILITY_IMMUNITY >= GEN_6 && IsElectricAbilityAffected(ABILITY_LIGHTNING_ROD))
```

Comment:
Is this intentionally 6 or should it be 5?

Reply:
Should be 5, thanks for the catch :)